index.html

```diff
@@ -151,18 +151,21 @@
     </p>
 
     <p>
-<a>Decentralized identifiers</a> (DIDs) are a new type of identifier to
-provide verifiable, decentralized digital identity. These new identifiers are
-designed to enable the controller of a <a>DID</a> to prove control over
-it and to be implemented independently of any centralized registry, identity
-provider, or certificate authority. <a>DIDs</a> are URLs that relate a
-<a>DID subject</a> to a <a>DID document</a> allowing trustable interactions with
-that subject. <a>DID documents</a> are simple documents describing how to use
-that specific <a>DID</a>. Each <a>DID document</a> can express cryptographic
-material, verification methods, or <a>service endpoints</a>, which provide a
-set of mechanisms enabling a <a>DID controller</a> to prove control of the
-<a>DID</a>. <a>Service endpoints</a> enable trusted interactions with the
-<a>DID subject</a>.
+<a>Decentralized identifiers</a> (DIDs) are a new type of identifier to 
+provide verifiable, decentralized digital identity. A <a>DID</a> may identify 
+any subject (e.g., a person, organisation, thing, datamodel, abstract entity, etc.)
```

Comment:
US spelling for W3C specs.

```suggestion
any subject (e.g., a person, organization, thing, data model, abstract entity, etc.)
```